src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-containerized-function-monitoring/enable-containerized-function.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: Create and deploy New Relic Lambda image layers
+metaDescription: After enabling New Relic's containerized Lambda function, you can create a layer as an image.
+redirects:
+freshnessValidatedDate: never
+---
+
+With containerized Lambda functions, you can create Lambda layers using container images. As an alternative to deploying layers as .zip archives, container images allow for increased flexibility and improved security.
+
+Before you get started, make sure you've built a Dockerfile that includes your chosen runtime environment, installed your dependencies, and copied them into the correct directory structure. 
+
+Once you've created the layer as an image, you'll push the image to an Amazon ECR repository. You will reference the image from Amazon ECR repository when you create your Lambda layer in the AWS Management Console. 
+
+First, you need to create your image layers. Once that's done, you can either create a regular or a public ECR function. To help you with this, we've created a [gallery of New Relic Lambda layers for Docker hosted on Amazon ECR](https://gallery.ecr.aws/x6n7b2o2?page=1). Alternatively, follow these steps to create your own.
+
+<Steps>
+<Step>
+### Create a New Relic Lambda layer as an image [#create-lambda-layer]
+
+Before you can deploy your image, you first need to create a New Relic Lambda layer. To do this, complete the following steps: 
+
+1. Choose from the different [Amazon Resource Name (ARN) layers available to you](https://layers.newrelic-external.com/#/get-layers/get) to find the one that matches your system.
+2. Create a new docker image using the following code:
+```
+FROM alpine:latest as layer-copy 
+# Create a build stage to copy the files from S3 using credentials ARG AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION:-"region"} 
+ARG AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID:-"accessKey"} 
+ARG AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY:-"secretkey"} ENV AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION} 
+ENV AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID} 
+ENV AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY} 
+RUN apk add aws-cli curl unzip 
+RUN mkdir -p /opt 
+RUN curl $(aws lambda get-layer-version-by-arn --arn <ARN Link> --query 'Content.Location' --output text) --output layer.zip 
+RUN unzip layer.zip -d /opt 
+RUN rm layer.zip 
+FROM scratch 
+# Start second stage from blank image to squash all previous history, including credentials. 
+WORKDIR /opt 
+COPY --from=layer-copy /opt .
+```
+3. Push that image to Amazon ECR. 
+
+```
+docker build -t layer-image1:latest .
+```
+</Step>
+
+<Step>
+### Deploy the image in AWS ECR [#deploy-image-ECR]
+
+Now that you've created the image, you can deploy it using the following steps: 
+
+1. Configure your AWS account from the terminal.
```

Comment:
Not sure what "configure" means here.

Reply:
Fair point. I added a link to the AWS CLI doc